tools/add_triage_signature.py

```diff
@@ -154,9 +165,9 @@ def _update_ticket(self, url, issue_key, should_update=False):
 
         url = self._logs_url_to_api(url)
         try:
-            md = self._get_metadata_json(url)
-        except:
-            logger.exception("Error getting logs for %s at %s", issue_key, url)
+            md = get_metadata_json(url)
+        except Exception as e:
```

Comment:
Can we move to `(requests.exceptions.ConnectionError, requests.exceptions.HTTPError)`?

Reply:
I prefer not to do this potentially breaking change in this PR